homework/02LessonTwo.js

```diff
@@ -126,7 +159,7 @@ function whichSpecies(character) {
 //
 // In your own words, explain what conditionals do.
 //
-// Your Answer Goes Here:
+// Your Answer Goes Here: Conditionals evaluate to true or false of a block of code though of series of conditionals (e.g. if, else).
 //
```

Comment:
Thank you soooo much! I fixed them all! I will need your help with the apps. Thanks again!

Sent from my iPhone

> On Aug 7, 2016, at 2:33 PM, maddiesolz <notifications@github.com> wrote:
> 
> In homework/02LessonTwo.js:
> 
> > @@ -126,7 +159,7 @@ function whichSpecies(character) {
> >  //
> >  // In your own words, explain what conditionals do.
> >  //
> > -// Your Answer Goes Here:
> > +// Your Answer Goes Here: Conditionals evaluate to true or false of a block of code though of series of conditionals (e.g. if, else).
> >  //
> Yep! I accept this answer
> 
> 
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>